src/vs/workbench/contrib/notebook/common/notebookOptions.ts

```diff
@@ -293,8 +336,13 @@ export class NotebookOptions extends Disposable {
 			configuration.interactiveWindowCollapseCodeCells = this.configurationService.getValue(NotebookSetting.interactiveWindowCollapseCodeCells);
 		}
 
+		if (outputLineHeight || fontSize || outputFontSize) {
+			configuration.outputLineHeight = this._computeOutputLineHeight();
+		}
+
 		this._layoutConfiguration = Object.freeze(configuration);
 
+		// add new configuration to the event?
```

Comment:
This reads a bit like a todo comment. Is action needed here?

Reply:
whoops, good catch (already done)